lib/retrieval-stats.js

```diff
@@ -137,14 +151,14 @@ const addHistogramToPoint = (point, values, fieldNamePrefix = '') => {
   }
 }
 
+// TODO: include Filecoin Deal id
+const getTaskId = (/** @type {import('./typings').Measurement} */m) => `${m.cid}`
```

Comment:
should this be `getDealId` now?

Reply:
We are using the term "task" in many other places. To change this, I have to review those places and decide whether switching from "tasks" to "deals" makes sense.

Let's defer this until later, perhaps as part of adding deal/miner info to our tasks (https://github.com/filecoin-station/roadmap/issues/65).